analyses/chromothripsis/04-plot-chromothripsis-and-breakpoint-data.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Chromothripsis Plots with Breakpoint Data'
+author: "Laura Egolf"
+date: "2021"
+output: html_document
+---
+  
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+This notebook plots the number of chromothripsis regions per sample along with breakpoint data from `analyses/chromosomal-instability/`. As expected, the number of chromothripsis calls roughly correlates with the breakpoint counts calculated from both SV and CNV data. 
+
+### Define directory paths and load packages
+
+```{r}
+# Set directory paths
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+analysis_dir <- file.path(root_dir, "analyses", "chromothripsis")
+plots_dir <- file.path(analysis_dir, "plots", "04-breakpoint-data")
+breakpoint_data_dir <- file.path(root_dir, "analyses", "chromosomal-instability", "breakpoint-data")
+
+# Create plots_dir if it doesn't exist
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir, recursive = TRUE)
+}
+
+# Load packages
+library(ggplot2)
+library(RColorBrewer)
+library(readr)
+library(dplyr)
+```
+
+
+### Prepare chromothripsis data and breakpoint data
+
+```{r}
+# Read in summary of chromothripsis regions per sample (number of regions for each confidence level)
+chromoth_per_sample <- read_tsv(file.path(analysis_dir, "results", "chromothripsis_summary_per_sample.txt"))
+
+# Read in CNV and SV breakpoint data
+cnv_densities <- read_tsv(file.path(breakpoint_data_dir, "cnv_breaks_densities.tsv"))
+sv_densities <- read_tsv(file.path(breakpoint_data_dir, "sv_breaks_densities.tsv"))
+
+# Rename columns before merging
+cnv_densities <- cnv_densities %>%
+  rename(Kids_First_Biospecimen_ID = samples) %>%
+  rename(cnv_breaks_count = breaks_count)
```

Comment:
I believe you could combine these `rename()` steps. Not particularly critical, though!
```suggestion
cnv_densities <- cnv_densities %>%
  rename(Kids_First_Biospecimen_ID = samples,
         cnv_breaks_count = breaks_count)
```

Reply:
Looks good to me!